cvat/apps/restrictions/models.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (C) 2020 Intel Corporation
```

Comment:
I believe accepted and declined agreements should be stored somewhere, but I can't find it.

Reply:
I'm not sure that we should store the agreements at the moment. For now if the user decline some of required agreement it cannot complete the registration and cannot continue to use the tool.
But I agree that this may be useful in the future.

Reply:
Isn't it required by some law?